src/app.js

```diff
@@ -1,4 +1,6 @@
 import './style.css';
+import { taskMarking, clearCompletedtask } from './task-mark-module.js';
+// import { clearCompletedtask } from './task-mark-module.js';
```

Comment:
- [ ] Please remove commented code to improve the quality of your code and make it cleaner